docs/CONTRIBUTING.md

```diff
@@ -2,9 +2,13 @@
 
 **This document is under review** 
 
+### Join the Signalen community mailinglist to stay up to date
+
+* Product owners for each development team coordinate in the product steering group. If you wish to contribute or keep up to date please join the [email list](https://lists.publiccode.net/mailman/postorius/lists/signalen-discuss.lists.publiccode.net/) and/or [Slack channel](https://samenorganiseren.slack.com/archives/C01AF8CN495).
+
 ### Want to add a new feature?
 
-* **At the moment we do not accept merge request with new features from anyone other than the core developers!**
+* See [signalen.org](https://signalen.org/en/using-signalen) for how to help with development or propose new features.
```

Comment:
What do we want to do with PR's from developers that want to add features and already have build them?

Reply:
I think there's a lot more which could be done to tighten up the wording in this page, and I propose we do that in steps. I'd like to see this much merged and then think about what more makes sense.

Regarding the PR questions specifically, that's up to you and the other backend devs to decide, of course. My guess is that any work you do which is not a part of your sprint is on a "best effort" basis, thus may not get any attention for a while, but would eventually be reviewed. But, I do not have a good proposal for how to formulate that guess as text for this file yet.